src/Components/Common/Animation/AnimationBar/AnimationBar.tsx

```diff
@@ -0,0 +1,41 @@
+import { type FC, type ReactElement, useEffect, useRef } from 'react';
+import classes from './AnimationBar.module.scss';
+import { randomBackground, randomHeight } from '../../../../utils/helpers/componentsHelpers';
+import { AnimationParameter } from '../../../../utils/types/enums';
+
+export const AnimationBar: FC = (): ReactElement => {
+  const barElement = useRef<HTMLDivElement>(null);
+
+  const barBackgrounds = [
+    AnimationParameter.RedBackground,
+    AnimationParameter.YellowBackground,
+    AnimationParameter.BlueBackground,
+    AnimationParameter.GreenBackground,
+  ];
+
+  const setBarStyles = (isInitial: boolean): void => {
+    if (barElement.current) {
+      barElement.current.style.background = randomBackground(barBackgrounds);
+      barElement.current.style.height = `${randomHeight(AnimationParameter.MaxHeight)}%`;
+      if (!isInitial) {
+        barElement.current.style.transition = AnimationParameter.Transition;
+      }
+    }
+  };
```

Comment:
We need to avoid style updates using ref If possible 
You can read details here: https://www.reddit.com/r/reactjs/comments/167yhvp/why_is_using_userefcurrent_to_change_the_styling/

Reply:
It is possible. I have done it!